### PR TITLE
mimir: use of CoordWrapper instead of Coord in Street and Addr

### PR DESCRIPTION
**Pull request by kadhikari**

### CODE REVIEW

File: libs/osm_builder/src/lib.rs [hunk 1]
```diff
@@ -107,14 +107,14 @@ impl OsmBuilder {
...
-    pub fn node(&mut self, coord: mimir::Coord, name: Option<String>) -> osmpbfreader::OsmId {
+    pub fn node(&mut self, coord: mimir::CoordWrapper, name: Option<String>) -> osmpbfreader::OsmId {
         if let Some(ref value) = name.clone().and_then(|n| self.named_nodes.get(&n)){
             return *value.clone()
         }
         let n = osmpbfreader::Node {
             id: self.node_id,
-            lat: coord.lat,
-            lon: coord.lon,
+            lat: coord.x,
```
Comment (TeXitoi) [inline libs/osm_builder/src/lib.rs]: I'd prefer `lat: coord.lat()`

File: libs/osm_builder/src/lib.rs [hunk 2]
```diff
@@ -107,14 +107,14 @@ impl OsmBuilder {
...
+            lon: coord.y,
```
Comment (TeXitoi) [inline libs/osm_builder/src/lib.rs]: same

File: libs/osm_builder/src/lib.rs [hunk 3]
```diff
@@ -33,8 +33,8 @@ extern crate osmpbfreader;
 extern crate mimir;
 use std::collections::BTreeMap;
 
-pub fn named_node(lat: f64, lon: f64, name: &'static str) -> (mimir::Coord, Option<String>) {
-    (mimir::Coord {lon: lon, lat: lat}, Some(name.to_string()))
+pub fn named_node(lat: f64, lon: f64, name: &'static str) -> (mimir::CoordWrapper, Option<String>) {
```
Comment (antoine-de) [inline libs/osm_builder/src/lib.rs]: maybe we can name it Coord ?

File: libs/osm_builder/src/lib.rs [hunk 4]
```diff
@@ -113,8 +113,8 @@ impl OsmBuilder {
         }
         let n = osmpbfreader::Node {
             id: self.node_id,
-            lat: coord.lat,
-            lon: coord.lon,
+            lat: coord.x,
+            lon: coord.y,
```
Comment (TeXitoi) [inline libs/osm_builder/src/lib.rs]: writing a Coord::lat() and Coord::lon() may be a good idea.

File: src/bin/osm2mimir.rs [hunk 5]
```diff
@@ -167,7 +167,7 @@ fn administrative_regions(pbf: &mut OsmPbfReader, levels: BTreeSet<u32>) -> Admi
                                            objects.get(&r.member).and_then(|value| {
                                                match value {
                                                    &osmpbfreader::OsmObj::Node(ref node) => {
-                                                       Some(mimir::CoordWrapper(geo::Coordinate {
+                                                       Some(mimir::Coord(geo::Coordinate {
```
Comment (TeXitoi) [inline src/bin/osm2mimir.rs]: better to use Coord::new

**Pull request merged**